OutOfSchool/OutOfSchool.WebApi/Util/QuartzPool.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using Quartz;
+
+namespace OutOfSchool.WebApi.Util
+{
+    /// <summary>
+    /// Represents a pool that contains default quartz tasks. All config actions will be configured by default quartz.
+    /// </summary>
+    public static class QuartzPool
+    {
+        private static List<Action<IServiceCollectionQuartzConfigurator>> quartzConfigActions
+            = new List<Action<IServiceCollectionQuartzConfigurator>>();
+
+        /// <summary>
+        /// Gets all quartz config actions.
+        /// </summary>
+        public static IReadOnlyList<Action<IServiceCollectionQuartzConfigurator>> GetQuartzConfigActions
+            => quartzConfigActions.AsReadOnly();
+
+        /// <summary>
+        /// Adds a job to the quartz pool.
+        /// </summary>
+        /// <param name="configure">Configures a given job.</param>
+        /// <typeparam name="T">Quartz job.</typeparam>
+        public static void AddJob<T>(Action<IJobConfigurator> configure = null)
```

Comment:
In case that method will be called into multiple threads `quartzConfigActions` will be broken as the `List` is not thread safe collection.
You can use `ConcurrentBag<>` instead.